src/automations/defaultActionValues.ts

```diff
@@ -68,8 +86,11 @@ export function getDefaultNotificationBody(template: AutomationTriggerTemplate):
       return NOTIFICATION_BODY_WORK_POOL_STATUS
     case 'work-queue-status':
       return NOTIFICATION_BODY_WORK_QUEUE_STATUS
+    case 'custom':
+      return NOTIFICATION_BODY_CUSTOM
     default:
-      throw new Error(`Default notification body missing for template type: ${template}`)
+      const exhaustiveCheck: never = template
+      throw new Error(`Default notification body missing for template type: ${exhaustiveCheck}`)
```

Comment:
weak opinion that this should be `return ''` instead of throwing a runtime error.
altho now that this exchaustiveCheck: never is here, we _should_ get issues at build-time now instead.